data/json/items/comestibles.json

```diff
@@ -6458,6 +6483,7 @@
     "nutrition": 1,
     "fun": 1,
     "flags": [ "EATEN_COLD" ],
+    "freezing_point": 23,
```

Comment:
https://acadianmaple.com/pages/all-about-maple-syrup
maple sap freezes a few degrees below water, but syrup freezes at -70F or so because of the sugar concentration. Sugar beet syrup and coffee syrup would have the same issue.